sdk/framework/backend.go

```diff
@@ -527,10 +528,12 @@ func (b *Backend) handleRootHelp(req *logical.Request) (*logical.Response, error
 	// the request, the type will be used as part of the request/response body
 	// names in the OAS document.
 	requestResponsePrefix := req.GetString("requestResponsePrefix")
+	genericPaths := fmt.Sprintf("%v", req.Data["genericPaths"])
+	generic, _ := strconv.ParseBool(genericPaths)
```

Comment:
I think it might be still worth specifying that it's "mount" paths (since we are dealing with many different paths in Vault).

Reply:
sounds good! If you think that's not too long. Do you think the query parameter should have `mount` as well? It seems a little long, but I don't have a strong opinion either way

Reply:
Why are we ignoring the error from `strconv.ParseBool()`?

Reply:
I just fixed the parsing bool to be more straightfoward